ruby_basics/11_nested_collections/spec/nested_array_exercises_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'spec_helper'
+require_relative '../exercises/nested_array_exercises'
+
+RSpec.describe 'Nested Array Exercises' do
+  describe 'blank seating chart creation exercise' do
+    it 'returns a 2d array (chart) containing 2 rows, each containing nil 3 times' do
```

Comment:
Personally I'm not in favour of this type of it description. The it should describe the behaviour of the test whereas you are describing an implementation detail.

The method doesn't return a 2d array containing 2 rows, each containing nil 3 items until you specify that as the input. If I specified a different input I wouldn't get that output. Generally I should be able to change the internals of the test without having to change the description of the test.

Reply:
Correct, this type of `it` description was extremely unnatural for me to do, but it is a pattern I saw in some of the other exercises and I liked the approach because it makes it very clear to the student what the test is checking.

I can see the counterarguments since the test shows what it is checking though. I'm totally open to revising these.